src/tools/FreehandMouseTool.js

```diff
@@ -1001,6 +1029,11 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
     element.addEventListener(EVENTS.MOUSE_DOWN, this._drawingMouseDownCallback);
     element.addEventListener(EVENTS.MOUSE_MOVE, this._drawingMouseMoveCallback);
 
+    element.addEventListener(EVENTS.TOUCH_START, this._drawingMouseMoveCallback);
```

Comment:
@dannyrb, should TOUCH_START really be mapped to the mouseMoveCallback? This isn't my area of expertise.

Reply:
i'm not quite sure why this is needed, but it only works that way at the moment

Reply:
This is all a little odd as `freehand` deviates from most other tools by having it's own "event loop". Because the tool is locked at this point, you're relying on the events emitted by cornerstone, where the dispatchers will continue to return prematurely because of the lock.

It looks like... `_activateDraw` is only called if we're trying to create new data.  I would need to trace these to see what's going on with these events, and why it only works when setup this way. 🤷‍♂️

Reply:
Maybe its just so that the lines get updated with mouse position? Feel free to make a separate method if binding all of `_drawingMouseMoveCallback` is overkill.

Reply:
With the recent changes to freehand it be better to use the same drag mechanism as mouse uses.